lib/evaluate.js

```diff
@@ -115,6 +115,7 @@ export const evaluate = async ({
     point.intField('round_index', roundIndex)
     point.intField('total_participants', Object.keys(participants).length)
     point.intField('total_measurements', measurements.length)
+    point.intField('total_instances', countStationInstances(measurements))
```

Comment:
Can we please unify the naming? Currently we use `node` in the spec and `instances` in this PR. Or do they mean something different?

Reply:
Hmm, I see what you mean!

Which name do you prefer to use going forward - `node` or `instance`?

I am leaning towards using "node", because that's what we use in our design docs.

Switching to "node" also requires less changes. Changing "maxTasksPerNode" would require us to deploy a new version of SPARK.

The small downside is that https://github.com/filecoin-station/spark-evaluate/pull/97 already introduced the name "instance", so I need to rework that part + update Grafana charts.

Reply:
Yeah I agree, `node` is nicer, especially since it's already in the spec